wrangler/src/ctrlAccounts/heal.js

```diff
@@ -1,14 +1,56 @@
-import { response } from 'cfw-easy-utils'
-import { heal } from '../@utils/trust'
+import { response } from 'cfw-easy-utils';
+import { StrKey } from 'stellar-base';
+import { heal } from '../@utils/trust';
 
 export default async ({ request, env }) => {
   const {
-    sourceaccount,
-    oldturret,
-    newturret,
-    functionhash
-  } = await request.json()
-
-  const healresponse = await heal(sourceaccount, oldturret, newturret, functionhash)
-  return response.json(healresponse)
-}
\ No newline at end of file
+    controlAccountId,
+    removeTurretAccountId,
+    newTurretAccountId,
+    txFunctionHash,
+    timestamp,
+    userAccountId,
+    fee,
```

Comment:
this is not documented in the api docs. Made a personal note to update this in the stellar-turrets-docs

Attempted to make these inputs more consistent for users so it was clearer what type of data was requested

Reply:
can we change timestamp to timeout